modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/OAuth2RestClient.java

```diff
@@ -406,6 +407,24 @@ private String buildRoleSearchEndpoint(String roleName, String audienceType, Str
         return filter.toString();
     }
 
+    /**
+     * Share the application with the organizations.
+     *
+     * @param appId                       The application ID.
+     * @param applicationSharePOSTRequest The application sharing details.
+     * @throws IOException Error when sharing the application.
+     */
+    public void shareApplication(String appId, ApplicationSharePOSTRequest applicationSharePOSTRequest) throws
+            IOException {
+
+        String jsonRequest = toJSONString(applicationSharePOSTRequest);
```

Comment:
This approach is fine. Alternatively could have load the request payload from a file and replace any valuve if required which needed to be edited.